packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -56,21 +59,22 @@ export function isConformant(
     autoControlledProps?: string[];
     deprecated_className?: string;
   },
-  options: Conformant = {},
+  options: Conformant,
 ) {
   const {
+    testPath,
     constructorName = Component.prototype.constructor.name,
     eventTargets = {},
     exportedAtTopLevel = true,
     hasAccessibilityProp = true,
     requiredProps = {},
     rendersPortal = false,
     wrapperComponent = null,
-    handlesAsProp = true,
```

Comment:
It might be better to bring back `handlesAsProp`, then within this file add the "as" tests to `disabledTests` if `handlesAsProp` is false.

Reply:
Open for discussion, but I'm not sure if this would be a change in the right direction. handleAsProp is only referenced once which is in the DropdownSearchInput test file. Since the previous "as" tests have been replaced with the new react-conformance tests, It might make more sense just to use the disabledTests prop and remove handleAsProp.

Reply:
Makes sense to use `disabledTests` for me